locales/en.default.schema.json

```diff
@@ -1407,6 +1407,21 @@
             "label": "Square"
           },
           "info": "Add images by editing your collections. [Learn more](https://help.shopify.com/en/manual/products/collections)"
+        },
+        "columns_desktop": {
+          "label": "Number of columns on desktop"
+        },
+        "header_mobile": {
+          "content": "Mobile Layout"
+        },
+        "columns_mobile": {
+          "label": "Number of columns on mobile",
+          "options__1": {
+            "label": "1 column"
+          },
+          "options__2": {
+            "label": "2 columns"
```

Comment:
Yea 🤔 Definitely. For now I just requested translations since they were approved. I think when we know theyll always be the same we can definitely do that